### PR TITLE
Include csv TEE

### PR DESCRIPTION
**Pull request by BaoshunFang**

### CODE REVIEW

File: src/api/Cargo.toml [hunk 1]
```diff
@@ -22,15 +22,18 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", tag = "v0.7.0"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, tag = "v0.7.0", optional = true}
+# TODO: change it to "0.8.0", once released.
+as-types = { git = "https://github.com/BaoshunFang/attestation-service.git", rev = "22a36a6"}
```
Comment (fenghao176) [inline src/api/Cargo.toml:26]: this should not be changed

**Comment by jialez0** [comment]
@BaoshunFang The current CI error is not caused by this PR, it is because we are currently updating the client tool.
For this PR, in order to make the CI to run successfully, you can modify here: https://github.com/confidential-containers/kbs/blob/main/tools/client/Cargo.toml#L23 . Lock the version of `kbs-protocol` to `dc15fdc7b5396b21a9b956dc1fb772d1df016f29`

File: src/api/Cargo.toml [hunk 2]
```diff
@@ -22,15 +22,18 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", tag = "v0.7.0"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, tag = "v0.7.0", optional = true}
+# TODO: change it to "0.8.0", once released.
+as-types = { git = "https://github.com/BaoshunFang/attestation-service.git"}
+# TODO: change it to "0.8.0", once released.
+attestation-service = { git = "https://github.com/BaoshunFang/attestation-service.git", default-features = false, optional = true}
```
Comment (jialez0) [inline src/api/Cargo.toml:28]: @BaoshunFang Now your AS PR is merged, we can keep this dependence upstream :)

**Review by jialez0 — APPROVED** [review]

File: src/api/Cargo.toml (hunk 2 above)
Comment (BaoshunFang) [reply src/api/Cargo.toml]: Done

**Pull request merged**